dev/RadioButtons/RadioButtons.idl

```diff
@@ -12,27 +12,30 @@ unsealed runtimeclass RadioButtons : Windows.UI.Xaml.Controls.Control
 
     Object ItemsSource;
     Windows.Foundation.Collections.IVector<Object> Items{ get; };
-    [MUX_DEFAULT_VALUE("winrt::RadioButtonsElementFactory{}")]
+    [MUX_DEFAULT_VALUE("winrt::make<RadioButtonsElementFactory>()")]
     Object ItemTemplate;
 
     Windows.UI.Xaml.UIElement ContainerFromIndex(Int32 index);
 
     [MUX_DEFAULT_VALUE("-1")]
     Int32 SelectedIndex;
-    Object SelectedItem{ get; };
+    Object SelectedItem;
     event Windows.UI.Xaml.Controls.SelectionChangedEventHandler SelectionChanged;
 
     [MUX_DEFAULT_VALUE("1")]
-    Int32 MaximumColumns;
+    Int32 MaxColumns;
+
     Object Header;
+    Object HeaderTemplate;
```

Comment:
>Object [](start = 4, length = 6)

This should be DataTemplate

Reply:
That would prevent a DataTemplateSelector from being set